platform/core/src/DICOMSR/SCOORD3D/utils/getRenderableData.js

```diff
@@ -1,9 +1,24 @@
 import csMath from 'cornerstone-math';
 import SCOORD_TYPES from '../constants/scoordTypes';
 
-const getRenderableData = (GraphicType, GraphicData, ValueType) => {
+const getRenderableData = (
+  GraphicType,
+  GraphicData,
+  ValueType,
+  imageMetadata
+) => {
   let renderableData;
 
+  // Question for Steve and Andrey, with study 1.3.6.1.4.1.14519.5.2.1.6279.6001.100063870746088919758706456900
```

Comment:
Can we remove this comment for Steve/Andrey and move it to PR comments instead of code?

Reply:
oh yes this has to be removed, thanks for catching it!

Reply:
uhm this commet was already removed in latest commit